modules/persistent-storage-local-create-cr.adoc

```diff
@@ -52,7 +52,7 @@ spec:
 <4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
 <5> The file system that is created when the local volume is mounted for the first time.
 <6> The path containing a list of local storage devices to choose from.
-<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.
+<7> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/disk/by-id/xxx`. PVs are created for these local disks when the provisioner is deployed successfully.
 +
```

Comment:
nit: lets use `wwn`  as placeholder for these disk names. They make bit more sense as disk ids - https://en.wikipedia.org/wiki/World_Wide_Name